src/math/clamm.rs

```diff
@@ -370,6 +370,12 @@ pub fn check_tick(tick_index: i32, tick_spacing: u32) -> TrackableResult<()> {
 
     Ok(())
 }
+pub fn calculate_min_amount_out(
```

Comment:
There should be some tests to this function as all function inside `clamm.rs` should be tested